opcua.cpp

```diff
@@ -1697,7 +1707,16 @@ OPCUA::Node::Node(uint32_t conn, const string &nodeId) : m_nodeID(nodeId)
 	readValue[2].indexRange = NULL;
 
 	int res;
-	if ((res = SOPC_ClientHelper_Read(conn, readValue, 3, values)) == 0)
+	int RETRY_COUNT=5;
+	while(RETRY_COUNT > 0)
+	{
+		res = SOPC_ClientHelper_Read(conn, readValue, 3, values);
+		if (res == 0)
+			break;
+		RETRY_COUNT++;
```

Comment:
This needs to be "_retry_count--_" As written, it will never stop retrying if the Read fails.